### PR TITLE
Systemd service

### PR DESCRIPTION
**Pull request by vinyll**
Permits to start and reload the server.
Hence to deploy with more stability.

### CODE REVIEW

**Comment by mlbiche** [comment]
Ok for me. Maybe the copy of the service to the systemd folder is missing somewhere ?

**Review by mlbiche — DISMISSED** [review]
Ok for me. Maybe the copy of the service to the systemd folder is missing somewhere ?

File: neomad.service [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ExecStart=/home/ubuntu/prod/venv/bin/gunicorn app -w 3 --timeout 60
+# Required on some systems
+WorkingDirectory=/home/ubuntu/prod
+Restart=always
+RestartSec=3
+# Output to syslog
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=neomad
+#User=<alternate user>
```
Comment (fuentesloic) [inline neomad.service]: As it's commented, why not removing it? same below

**Review by fuentesloic — APPROVED** [review]

**Review by mlbiche — APPROVED** [review]
LGTM, still the same comment about the service copy to systemd folder ?

**Comment by vinyll** [comment]
> LGTM, still the same comment about the service copy to systemd folder ?

It's a symlink rather than a copy here.
The whole process for installation in production is not set in the makefile, it's hand made once with sudo, so it would be a different mission.

**Pull request merged**